chain/network/src/types.rs

```diff
@@ -457,6 +458,8 @@ pub struct Consolidate {
     pub(crate) other_edge_info: EdgeInfo,
     // Protocol version of new peer. May be higher than ours.
     pub(crate) peer_protocol_version: ProtocolVersion,
+    /// A helper data structure for limiting reading, reporting bandwidth stats.
```

Comment:
nit: is it a 'data structure' ? ;-) or simply a class

Reply:
@mm-near Rust doesn't support classes: https://stevedonovan.github.io/rust-gentle-intro/object-orientation.html